pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -429,7 +429,8 @@ func (r *ReconcileScanSettingBinding) verifyRoleInScanSettingBinding(
 
 		profileType, err := getScanType(tp.GetAnnotations())
 		if err != nil {
-			return fmt.Errorf("error getting profile type %s: %w", prfRef.Name, err)
+			logger.Info("error getting scan type, assuming Platform", "error", err)
+			profileType = compliancev1alpha1.ScanTypePlatform
```

Comment:
Should the annotations be added, too?

Reply:
I don't think so, it doesn't seem like something that the controller should do in the first place

Reply:
you have a point here though, we could go one level up and look at the annotation of the profile itself that the TP extends (if any).

Reply:
done